component/authenticator/pom.xml

```diff
@@ -159,7 +159,7 @@
                             javax.servlet.http,
                             org.wso2.carbon.extension.identity.helper.*;
                             version="${identity.extension.utils.import.version.range}",
-                            org.wso2.carbon.identity.governance.service.notification;
+                            org.wso2.carbon.identity.governance.*;
```

Comment:
Shouldn't we keep the old range imported avoid issues we faced while changing framework version ?

Reply:
Kept the old version range [1.1.37, 2.0.0). Changed this since there are two missing governance packages.